src/app/shared/employement/tf-employement.component.scss

```diff
@@ -0,0 +1,11 @@
+@import './../sass/_index';
```

Comment:
partial files can be imported without the underscore
for eg: you can and should write this as `@import './../sass/index`